tests/checker/httpserver.py

```diff
@@ -49,10 +49,11 @@ def log_message(self, format, *args):
         """
         pass
 
+
 # serve .xhtml files as application/xhtml+xml
-StoppableHttpRequestHandler.extensions_map.update({
-        '.xhtml': 'application/xhtml+xml',
-})
+StoppableHttpRequestHandler.extensions_map.update(
+    {".xhtml": "application/xhtml+xml"}
+)
```

Comment:
Did Black do this?  Weird.  I'd've gone with

```suggestion
StoppableHttpRequestHandler.extensions_map.update({
    ".xhtml": "application/xhtml+xml",
})
```

Reply:
Hi, _Black_ contributor here. Yeah, _Black_ did that, _Black_ will unpack multiple brackets. 

Pulled from https://github.com/psf/black/issues/177#issuecomment-385201073
> As for unpacking multiple brackets to separate indentation level, Black will continue doing this. I don't like multiple unrelated brackets shoved together to save a line.

Reply:
Ah, thank you for clarifying!

I like multiple unrelated brackets shoved together, not to save a line, but to save an indentation level.